qt_ui/windows/basemenu/airfield/QAircraftRecruitmentMenu.py

```diff
@@ -92,25 +92,20 @@ def buy(self, unit_type):
                     self, "No space for additional aircraft",
                     f"There is no parking space left at {self.cp.name} to accommodate another plane.", QMessageBox.Ok)
                 return
-
+                
         super().buy(unit_type)
         self.hangar_status.update_label()
 
     def sell(self, unit_type: UnitType):
         # Don't need to remove aircraft from the inventory if we're canceling
         # orders.
-        if self.pending_deliveries.units.get(unit_type, 0) <= 0:
-            global_inventory = self.game_model.game.aircraft_inventory
-            inventory = global_inventory.for_control_point(self.cp)
-            try:
-                inventory.remove_aircraft(unit_type, 1)
-            except ValueError:
-                QMessageBox.critical(
-                    self, "Could not sell aircraft",
-                    f"Attempted to sell one {unit_type.id} at {self.cp.name} "
-                    "but none are available. Are all aircraft currently "
-                    "assigned to a mission?", QMessageBox.Ok)
-                return
+        if self.pending_deliveries.units.get(unit_type, 0) <= 0 - self.cp.base.total_units_of_type(unit_type):
```

Comment:
`if pending_deliveries + current_units <= 0` is much clearer IMO.

Reply:
Fixed - this was no longer needed with the old check reimplemented anyway.